### PR TITLE
111rc2

### PR DESCRIPTION
**Pull request by dustinxie**
1. split testnet instruction into separate docs (README_testnet.md, README_CN_testnet.md)
2. pre-release v1.11 for testnet

### CODE REVIEW

**Review by Liuhaai — APPROVED** [review]

**Review by Liuhaai — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -8,25 +8,20 @@
 - [Release Status](#status)
 - [Join MainNet](#mainnet)
 - [Join Mainnet without using Docker](#mainnet_native)
-- [Join TestNet](#testnet)
```
Comment (CoderZhi) [inline README.md:11]: keep it

File: README.md [hunk 2]
```diff
@@ -8,25 +8,20 @@
...
-- IoTeX Mainnet: v1.10.1
-- IoTeX Testnet: v1.10.1
+- IoTeX MainNet: v1.10.1
 
-<!--
-- MainNet: v1.9.2
-- TestNet: v1.9.2
--->
+**Note**
+To start and run a testnet node, please click [**Join Testnet**](https://github.com/iotexproject/iotex-bootstrap/blob/master/README_testnet.md)
```
Comment (CoderZhi) [inline README.md:24]: ## <a name="testnet"/>Join TestNet
To start and run a testnet node, please click [**Join Testnet**](https://github.com/iotexproject/iotex-bootstrap/blob/master/README_testnet.md)

File: README_CN.md [hunk 3]
```diff
@@ -7,20 +7,20 @@
 - [发布状态](#status)
 - [加入主网](#mainnet)
 - [不使用Docker加入主网](#mainnet_native)
-- [加入测试网络](#testnet)
```
Comment (CoderZhi) [inline README_CN.md:10]: keep it

File: README_CN.md [hunk 4]
```diff
@@ -7,20 +7,20 @@
...
 ## <a name="status"/>发布状态
 
 以下是当前我们使用的软件版本：
 
-- 主网：v1.10.0
-- 测试网：v1.10.0
+- 主网：v1.10.1
+
+**Note**
+如果你要启动节点加入测试网，请点击[**加入测试网**](https://github.com/iotexproject/iotex-bootstrap/blob/master/README_CN_testnet.md)
```
Comment (CoderZhi) [inline README_CN.md:23]: same here

**Review by CoderZhi — DISMISSED** [review]

**Review by xianhuawei — APPROVED** [review]

**Review by CoderZhi — APPROVED** [review]

**Pull request merged**